Source/Add-ins/Profile.Common/ModalWindows/PointsListModalWindow.cs

```diff
@@ -16,35 +17,36 @@ public partial class PointsListModalWindow : Form
         public IPoint SelectedPoint;
         private Dictionary<int, IPoint> _points = new Dictionary<int, IPoint>();
 
-        public PointsListModalWindow(Dictionary<int, IPoint> points, string layer)
+        public PointsListModalWindow(Dictionary<int, IPoint> points)
         {
             InitializeComponent();
             InitializeListView(points);
 
-            lblLayer.Text = layer;
+            lblLayer.Text = LocalizationContext.Instance.FindLocalizedElement("LblGetPointsFromGeoCalculator", "Список точок модулю Геокалькулятор");
             _points = points;
         }
 
         private void InitializeListView(Dictionary<int, IPoint> points)
         {
             lvPoints.View = View.Details;
 
-            lvPoints.Columns.Add("Number", (int)(lvPoints.Width * 0.3));
-            lvPoints.Columns.Add("X", (int)(lvPoints.Width * 0.3));
-            lvPoints.Columns.Add("Y", (lvPoints.Width - (lvPoints.Columns[0].Width + lvPoints.Columns[1].Width)- 25));
-
-            lvPoints.HeaderStyle = ColumnHeaderStyle.Nonclickable;
-
+           
             lvPoints.Items.Clear();
 
             foreach(var point in points)
             {
                 var item = new ListViewItem(point.Key.ToString());
-                item.SubItems.Add(point.Value.X.ToString());
-                item.SubItems.Add(point.Value.Y.ToString());
+                item.SubItems.Add(Math.Round(point.Value.X, 5).ToString());
```

Comment:
All coordinates should have point as the decimal delimiter, regardless of the delimiter defined in the current culture 
There should be a method to return thew replaced string value (might be a extension for IPoint) 
Following is an example  Replace(",", CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator).Replace(".", CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator);